src/CodeGen.fs

```diff
@@ -1073,9 +1097,11 @@ let sampleFSharpNewtonsoftClientMember query queryName hasVariables useTasks =
         {builder} {{
             let query = {query}
 
-            let inputJson = JsonConvert.SerializeObject({body}, settings)
             {requestBody}
-            let responseJson = JsonConvert.DeserializeObject<JObject>(responseContent, settings)
+            let! responseContent = Async.AwaitTask(response.Content.ReadAsStreamAsync())
```

Comment:
```suggestion
            let! responseContent = response.Content.ReadAsStreamAsync() |> Async.AwaitTask
```
@Zaid-Ajaj, I propose to use pipes for all `Async.AwaitTask` and `Async.RunSynchronously` calls as it makes code more readable

Reply:
I am not a big fan of using `|>` for a single function call, let's keep it as is for now please